frontend/src/views/SignUp.vue

```diff
@@ -1,19 +1,30 @@
 <template>
   <div class="auth-page">
-    <AppAuthForm v-if="!loading"
+    <AppAuthForm v-if="!loading && !isFirebaseAuthorised"
```

Comment:
Same same.

Reply:
hnnngg yeah okay i'll split it to an onboarding view i guess?